source/guide_mumble.rst

```diff
@@ -131,7 +121,7 @@ Create ``~/etc/services.d/mumble.ini`` with the following content:
 .. code-block:: ini
 
   [program:mumble]
-  command=%(ENV_HOME)s/mumble/murmur.x86 -fg -ini %(ENV_HOME)s/mumble/murmur.ini
+  command=%(ENV_HOME)s/mumble/murmur.x86_64 -fg -ini %(ENV_HOME)s/mumble/murmur.ini
```

Comment:
This would only work with an executable file, but the murmur.x86_64 from your zip archive is not executable. So you need a 'chmod +x ~/mumble/murmur.x86_64' to make it work.

Reply:
addressed with https://github.com/Uberspace/lab/pull/1456/commits/e536d242751a851fe06bf07e5d22bf4c41d06a75